.github/workflows/docs.yml

```diff
@@ -23,3 +24,108 @@ jobs:
         with:
           path: ./docs/_build/html/
           name: docs_build
+
+  build_docs:
+    # This build is to check links and update table of supported formats
+    name: Check links and update table
+    runs-on: ubuntu-latest
+    env:
+      DOCS_PATH: ./docs/_build/html/
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge
+          channels: conda-forge
+          channel-priority: strict
+          environment-file: conda_environment.yml
+          activate-environment: rosettasciio
+
+      - name: Install build docs
+        shell: bash -l {0}
+        run: |
+          mamba install pydata-sphinx-theme
+
+      - name: Conda info
+        shell: bash -l {0}
+        run: |
+          conda info
+          conda list
+
+      - name: Update packages list
+        shell: bash -l {0}
+        run: |
+          # python generate_supported_format_table.py
+          echo "Table of Supported Format"
+          echo "========================="
```

Comment:
```suggestion
          echo "Table of Supported Formats"
          echo "=========================="
```

Reply:
However, the file currently already contains a headline. Idea is to create a script for autocompilation in the future?

Reply:
Yes, this is the idea!